build.gradle

```diff
@@ -162,3 +148,9 @@ idea {
         languageLevel = '11'
     }
 }
+
+tasks.each {
+  task -> if (task.hasProperty("duplicatesStrategy")) {
```

Comment:
What does that do?

Reply:
this is just an annoyance, sometimes there are resources with the same name especially in tests and gradle complains. we have this same fix in atlas too 🤷‍♀️